editor/src/components/editor/editor-modes.ts

```diff
@@ -176,10 +177,11 @@ export const EditorModes = {
       selectOnFocus: selectOnFocus,
     }
   },
-  commentMode: function (location: CanvasPoint | null): CommentMode {
+  commentMode: function (location: CanvasPoint | null, isDragging: boolean): CommentMode {
```

Comment:
nit: isDragging could be of type `'dragging' | 'not-dragging'`

Reply:
ok, see 323f73f197942d609b6bc82dd5826be5b741990c